tests/system/fim/common_tasks/delete_files.py

```diff
@@ -12,8 +12,25 @@
 import logging
 import os
 import time
-
```

Comment:
Two spaces missing here, between imports and function declaration. Try ` flake8 --max-line-length=100 delete_files.py`

Reply:
Ok, thanks! https://github.com/wazuh/wazuh-qa/pull/570/commits/e2936dcafee9410d734ae16cb532ceb72e621bcc